CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.3.1
+* **[Feature]**: Error handling implementation. - [PR #475](https://github.com/microsoft/vscode-edge-devtools/pull/475)
+* **[Feature]**: Implement CSS mirror editing - [PR #462](https://github.com/microsoft/vscode-edge-devtools/pull/462)
+* **[Bug Fix]**: Properly resolve index.html and other entrypoint links - [PR #469](https://github.com/microsoft/vscode-edge-devtools/pull/469)
+* **[Feature]**: Introducing error reporter. - [PR #474](https://github.com/microsoft/vscode-edge-devtools/pull/474)
+* **[Bug Fix]**: Supporting following CSS links on WSL remotes - [PR #467](https://github.com/microsoft/vscode-edge-devtools/pull/467)
+* **[Feature]**: Update min supported version for CDN tools - [PR #466](https://github.com/microsoft/vscode-edge-devtools/pull/466)

```

Comment:
Can we remove the last 2? they were included in the last release as part of the CDN system, and don't need to be called out specifically here